components/movementsensor/mpu6050/mpu6050.go

```diff
@@ -0,0 +1,291 @@
+// Package mpu6050 implements the movementsensor interface for an MPU-6050 6-axis accelerometer.
+package mpu6050
+
+import (
+	"context"
+	"encoding/binary"
+	"sync"
+
+	"github.com/edaniels/golog"
+	"github.com/golang/geo/r3"
+	geo "github.com/kellydunn/golang-geo"
+	"github.com/pkg/errors"
+	"go.viam.com/utils"
+
+	"go.viam.com/rdk/components/board"
+	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/components/movementsensor"
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/registry"
+	"go.viam.com/rdk/spatialmath"
+	rutils "go.viam.com/rdk/utils"
+)
+
+const modelName = "mpu6050"
```

Comment:
[nit] consider giving the model name a prefix `imu-mpu6050` or `gyro-mpu6050`

Reply:
Rand and I discussed: there are lots of types of movement sensors, and the extra prefix will distinguish this one from a GPS-based or camera-based movement sensor.